### ISSUE TITLE
Enhancement Request: Ability for More Styling for Corners

### ISSUE THREAD

**Issue by ablankstory** [kozakdenys/qr-code-styling#173]
Great project!

I was wondering if more options could be added for corner square styling:

1. Different weights for the stroke lines for the borders (Marker Borders)
2. Rounded corners for the Market Centers

Current
<img width="111" alt="Screenshot 2023-04-27 at 8 28 03 AM" src="https://user-images.githubusercontent.com/104391159/234876639-33240d7c-8ceb-453d-b090-d6c987bb5468.png">

Requested
<img width="86" alt="Screenshot 2023-04-27 at 8 28 25 AM" src="https://user-images.githubusercontent.com/104391159/234876737-05196da6-48da-431d-b8c8-1404467e1588.png">

Thanks for the consideration!

**Comment by nicoschuck** [comment]
I agree, it would be very helpful to get more options for the corner dots. An option for each border radios would be helpful as well. So that the user can decide which corner is rounded and which not: 
<img width="161" alt="image" src="https://github.com/user-attachments/assets/235168b8-6e29-47db-b516-f1da6b4ad0b3">

**Comment by vnkapoor** [comment]
Added same suggestion [here](https://github.com/kozakdenys/qr-code-styling/issues/269)

**Comment by chrisspiegl** [comment]
This seems to be getting worked on in (and appears to be a duplicate of):
- #46 

With pull request:
- https://github.com/kozakdenys/qr-code-styling/pull/274
